keanu-project/src/main/java/io/improbable/keanu/tensor/bool/SimpleBooleanTensor.java

```diff
@@ -304,6 +304,14 @@ public int hashCode() {
         return result;
     }
 
+    @Override
+    public String toString() {
+        return "SimpleBooleanTensor{" +
+            "data=" + Arrays.toString(data) +
```

Comment:
What happens when `data` is massive? I think we should curtail it at a fixed number of entries.

Reply:
Good point. I've limited it to the first 10 and last 10 of the buffer.